dspace-spring-rest/src/test/java/org/dspace/app/rest/builder/AbstractDSpaceObjectBuilder.java

```diff
@@ -106,6 +107,60 @@ protected AbstractDSpaceObjectBuilder(Context context) {
         return (B) this;
     }
 
+    protected <B extends AbstractDSpaceObjectBuilder<T>> B setRemovePermissionForEperson(DSpaceObject dso,
```

Comment:
this is not necessary as the test can be done using an administrator user. Nevertheless, it is a good idea to introduce support for add/remove permissions but it will require more investigation as we need to be sure that these extra permissions (resourcepolicies) are also cleanup when the test end. It will be probably easier to delegate to a ResourcePolicy builder for that.

Reply:
should I remove this code now or leave it in place for further investigation? It sounds like we should remove it, but checking to be sure.